webpack.config.js

```diff
@@ -29,7 +29,10 @@ module.exports = {
   output: {
     filename: 'materialize.js',
     path: path.resolve(__dirname, 'bin'),
-    libraryTarget: 'var',
-    library: 'M'
+    libraryTarget: 'umd',        
+    globalObject: 'this'
+  }, 
+  optimization: {
+    minimize: false,
   }
```

Comment:
Why is that? I guess because of the function name obfuscation due to minimization process.